councilmatic_core/management/commands/make_fixtures.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+import django
+from django.conf import settings
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+
+class Command(BaseCommand):
```

Comment:
it was a massive pita to refresh the test fixtures. this is a convenience command leveraging [my fork of django-fixture-magic](https://github.com/hancush/django-fixture-magic/tree/fix-merge) to dump the core models, and their relations. i use this, rather than dump data, because it allows you to dump a subset of your database and automatically grabs related items. dumping the full instance db resulted in an enormous fixture file that exponentially increased test runtime.